include/arch/arc/v2/linker_mwdt.ld

```diff
@@ -83,6 +83,7 @@ SECTIONS {
  */
 #include <snippets-text-start.ld>
 		*(".text")
+		*(".text*")
```

Comment:
It's cleaner to do

*(.text .text* .text.*)

Then the linker is not forced to place these input sections after each other and has more freedom to optimize.